lizzy/api.py

```diff
@@ -4,14 +4,15 @@
 from typing import (Dict, List, Optional,  # noqa pylint: disable=unused-import
                     Tuple)
 
+import yaml
 from decorator import decorator
 
 import connexion
-import yaml
 from flask import Response
-from lizzy import config
+from lizzy import config, metrics
 from lizzy.apps.senza import Senza
 from lizzy.exceptions import ExecutionError, ObjectNotFound, TrafficNotUpdated
+from lizzy.metrics import MeasureRunningTime
```

Comment:
So you suggest to use `metrics.MeasureRunningTime("..")`? I also found it easier to read without the prefix `metrics.*`. Or I can change the name from `MeasureRunningTime` to `RunningTime` and then use `metrics.RunningTime("..")` which seems also cool.